docs/tutorials/geohash-substrings.rst

```diff
@@ -169,7 +169,7 @@ This discussion of the algorithm is coarse, in part because its role in
 the overall query process has been simplified to streamline the
 explanation in this short note. The full version as used within the
 query planner can be found nested inside the
-`getUniqueGeohashSubstringsInPolygon <https://github.com/locationtech/geomesa/blob/master/geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geohash/GeohashUtils.scala#L914>`__
+`getUniqueGeohashSubstringsInPolygon <https://github.com/locationtech/geomesa/blob/master/geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geohash/GeohashUtils.scala#L937>`__
```

Comment:
yeah, on the main github page just click on the 'branch' drop down, pick the 'tags' side, and pick the m0 branch. The end result is https://github.com/locationtech/geomesa/tree/geomesa_2.11-1.3.0-m0. from there, browse to the file you need, click on the left side to indicate the line numbers, and then copy the url from the browser